app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,21 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time     = options[:time] ? "time > \"#{options[:time]}\"" : 'time > yesterday'
+    @query    = "#{options[:query]} and #{time}"
+    @limit   = options[:limit] || 50
+    @audits  = Audit.search_for(@query).limit(@limit)
+    @count   = Audit.search_for(@query).count
+    @subject = _("Audit summary")
+
+    set_url
+    set_locale_for(user)
+
+    mail(:to      => user.mail,
+         :from    => Setting["email_reply_address"],
+         :subject => @subject,
+         :date    => Time.now)
```

Comment:
Timezone's irrelevant really, it's a generated header that the MUA will convert on display.